code/game/machinery/computer/crew.dm

```diff
@@ -33,9 +33,17 @@
 		src.updateDialog()
 		return
 
+var/datum/singleton/crewmonitor/crewmonitor = new

```

Comment:
To avoid confusion: I'm referring to the use of "singleton" in the path name. IIRC the use of "static" or "global" between "/var" and "/datum" is an instruction for BYOND and not part of the type path.